### PR TITLE
User impersonation copyedit and new images added

### PR DESCRIPTION
**Pull request by jeffreylees**

### CODE REVIEW

File: articles/user-profile/user-impersonation.md [hunk 1]
```diff
@@ -64,11 +65,11 @@ Let's assume that you have two apps, `app1` and `app2`, and you want to imperson
 
 ### Get a Token
 
-First, you have to generate a token that you can use to call the [Management APIv2](/api/management/v2), specifically the [Impersonation endpoint](/api/authentication/reference#impersonation). This token is called __Auth0 Management APIv2 Token__.
+First, you have to generate a token that you can use to call the [Management APIv2](/api/management/v2), specifically the [Impersonation endpoint](/api/authentication/reference#impersonation). This token is called the Auth0 Management APIv2 Token.
```
Comment (mpaktiti) [inline articles/user-profile/user-impersonation.md]: The Impersonation endpoint is part of the Authentication API, not Management APIv2

File: articles/user-profile/user-impersonation.md [hunk 2]
```diff
@@ -2,57 +2,58 @@
...
+::: panel Acquiring a Token
+Impersonating a user using the [Dashboard](${manage_url}) will not return an [ID Token](/tokens/id-token) to your application by default. There are two ways to achieve this. You can alter the "Response Type" setting in the impersonation menu's [Advanced Settings](#advanced-settings) from "Code" to "Token" (Sign in as user -> Show Advanced Settings). Alternatively, you can add `additionalParameters.scope: "openid"` to the request body while calling the [impersonation endpoint](/api/authentication/reference#impersonation) manually.
 :::
 
 ### Advanced Settings
 
-When impersonating a user in Dashboard, after clicking __Sign in as User__ you will see a link to expand __Advanced Settings__.
+When impersonating a user in Dashboard, after clicking "Sign in as User" you will see a link to expand "Advanced Settings".
```
Comment (mpaktiti) [inline articles/user-profile/user-impersonation.md]: Can you revert this? Use bold for field/button/screen names.

File: articles/user-profile/user-impersonation.md [hunk 3]
```diff
@@ -2,57 +2,58 @@
...
 
-![Links for User Impersonation](/media/articles/user-profile/signin-as-user-02.png)
+![Links for User Impersonation](/media/articles/user-profile/user3.png)
 
-::: note
-Impersonating a user using the Dashboard will not return a [ID Token](/tokens/id-token) to your application by default. To achieve this, call the [impersonation endpoint](/api/authentication/reference#impersonation) manually or in the [Advanced Settings](#advanced-settings). If you call the endpoint manually, add `additionalParameters.scope: "openid"` to the request body.
+::: panel Acquiring a Token
+Impersonating a user using the [Dashboard](${manage_url}) will not return an [ID Token](/tokens/id-token) to your application by default. There are two ways to achieve this. You can alter the "Response Type" setting in the impersonation menu's [Advanced Settings](#advanced-settings) from "Code" to "Token" (Sign in as user -> Show Advanced Settings). Alternatively, you can add `additionalParameters.scope: "openid"` to the request body while calling the [impersonation endpoint](/api/authentication/reference#impersonation) manually.
```
Comment (mpaktiti) [inline articles/user-profile/user-impersonation.md]: Use bold for field names, code formatting for values, ie: You can alter the __Response Type__ setting in the impersonation menu's [Advanced Settings](#advanced-settings) from `Code` to `Token` (__Sign in as user__ > __Show Advanced Settings__)

File: articles/user-profile/user-impersonation.md [hunk 4]
```diff
@@ -2,57 +2,58 @@
...
 
 ::: note
-  Any Rules that you've implemented will run when you impersonate a user, including any actions that update the user.
+Any Rules that you've implemented will run when you impersonate a user, including any actions that update the user.
 :::
 
 ## Use the Dashboard
 
-Navigate to the [Users](${manage_url}/#/users) page in the Management Dashboard and select the user you want to login as. Click on the __Sign in as User__ and select the client you want to log into using the dropdown menu.
+Navigate to the [Users](${manage_url}/#/users) page in the Management Dashboard and select the user you want to login as. Click on the "Sign in as User: and select the client you want to log into using the dropdown menu.
```
Comment (mpaktiti) [inline articles/user-profile/user-impersonation.md]: Revert to bold since this is a field

File: articles/user-profile/user-impersonation.md [hunk 5]
```diff
@@ -2,57 +2,58 @@
...
 
 Auth0 provides a _Sign in As_ feature for user impersonation, and provides the following:
+
 - Detailed auditing of who impersonated when.
 - Restrictions on impersonation which allows you to reject an impersonated authentication transaction based on, for instance, corporate policies around privacy and sensitive data.
 - Unlimited customization on who can impersonate who, when, depending on whatever context, using our [Rules](/rules) engine. In a Rule, you have access to `user.impersonated` (the impersonated login) and `user.impersonator` (the impersonating login) and you can write arbitrary Javascript to define how it works.
 
 ::: note
-  Any Rules that you've implemented will run when you impersonate a user, including any actions that update the user.
+Any Rules that you've implemented will run when you impersonate a user, including any actions that update the user.
```
Comment (mpaktiti) [inline articles/user-profile/user-impersonation.md]: Add link to rules :)

File: articles/user-profile/user-impersonation.md [hunk 6]
```diff
@@ -2,57 +2,58 @@
 description: This page explains how to impersonate a user, often used for testing and troubleshooting purposes.
 toc: true
 ---
-
 # User Impersonation
 
-::: panel-warning Advanced Feature
-Impersonation functionality may be disabled by default for your tenant. To check, go to the [Users](${manage_url}/#/users) page in the Dashboard, select a user, and see if the __Sign in as User__ button is displayed. If you can't see it, [contact support](${env.DOMAIN_URL_SUPPORT}) and ask them to enable the feature for your tenant.
+::: warning
+Impersonation functionality may be disabled by default for your tenant. To check, go to the [Users](${manage_url}/#/users) page in the Dashboard, select a user, and see if the "Sign in as User" button is displayed. If you can't see it, [contact support](${env.DOMAIN_URL_SUPPORT}) and ask them to enable the feature for your tenant.
```
Comment (mpaktiti) [inline articles/user-profile/user-impersonation.md]: Revert to bold since this is a field

**Review by mpaktiti — CHANGES_REQUESTED** [review]
Hey, I left some comments

**Comment by jeffreylees** [comment]
@mpaktiti I rewrote/rearranged a good portion of the sample section, as the wording made not a lot of sense to me. Can you take a glance at the changed parts and let me know what you think?

**Review by mpaktiti — APPROVED** [review]
Looks good Jeff! 👍

**Pull request merged**